stripe/src/main/java/com/stripe/android/util/StripeTextUtils.java

```diff
@@ -81,6 +82,107 @@ public static boolean isBlank(String value) {
     }
 
     /**
+     * Returns a {@link CardBrand} corresponding to a partial card number, or {@link Card#UNKNOWN}
+     * if the card brand can't be determined from the input value.
```

Comment:
Hmm. It would make sense to return all the valid types in this function, and then if I wanted to only change the UI when one of them is available, I could do that outside this class.